### ISSUE TITLE
Missing String values

### ISSUE THREAD

**Issue by Linfye** [scribe-org/Scribe-i18n#74]
### Terms

- [X] I have searched [open and closed issues](https://github.com/scribe-org/Scribe-i18n/issues?q=is%3Aissue)
- [X] I agree to follow Scribe-i18n's [Code of Conduct](https://github.com/scribe-org/Scribe-i18n/blob/main/.github/CODE_OF_CONDUCT.md)

### Issue

After running `git subtree pull --prefix app/src/main/assets/i18n git@github.com:scribe-org/Scribe-i18n.git main --squash` to pull the latest i18n, the compiler reported the following errors:
```
Android resource linking failed
warn: removing resource be.scri.debug:string/app.about.legal.third_party.author without required default value.
warn: removing resource be.scri.debug:string/app.about.legal.third_party.license without required default value.
warn: removing resource be.scri.debug:string/app.about.legal.third_party.link without required default value.
warn: removing resource be.scri.debug:string/app.download.menu_option.download_data without required default value.
be.scri.app-mergeCoreDebugResources-57:/layout/fragment_main.xml:164: error: resource string/app.download.menu_option.download_data (aka be.scri.debug:string/app.download.menu_option.download_data) not found.
error: failed linking file resources.
```
It appears that some String values might miss.

**Comment by andrewtavis** [comment]
CC @henrikth93 and @Jag-Marcel :) Would we be able to get the Swedish and German localizations updated?

**Comment by andrewtavis** [comment]
Feel free to work on the Android corner switching as well as you can without these, @Linfye!

**Comment by henrikth93** [comment]
> CC @henrikth93 and @Jag-Marcel :) Would we be able to get the Swedish and German localizations updated?

Sure, is it just to start translating it?

**Comment by andrewtavis** [comment]
Yes, that would be great. We added in the rest of the strings from Figma, so now there's more to localize than when you first worked on it :)

**Comment by andrewtavis** [comment]
Closed by #75. Thanks @henrikth93 and @Jag-Marcel! 🚀
